db/models/assessment.py

```diff
@@ -59,3 +64,18 @@ def register_application(application_id: str):
                 message="An assessment for this application already exists"
             )
         return assessment
+
+    @staticmethod
+    def update_status(assessment_id: str, status: str):
+        try:
+            assessment_id = AssessmentMethods.get_by_id(assessment_id)
+            assessment_id.compliance_status = status
+            db.session.commit()
+            assessment = Assessment(
+                application_id=assessment_id.application_id,
+                id=assessment_id.id,
+                compliance_status=assessment_id.compliance_status,
+            )
+            return assessment
+        except AttributeError:
+            raise AssessmentError(message="An assessment id doesn't not exist")
```

Comment:
typo
```suggestion
            raise AssessmentError(message="An assessment does not exist")
```

Reply:
Thank you 😄

Reply:
I think this try catch is not adding any value, as if the ID was not found it already would have thrown an exception here: https://github.com/communitiesuk/funding-service-design-assessment-store/blob/b84c92dc27864fd1749314fde47795fe4b84f738/db/models/assessment.py#L52

Reply:
Yeah, I agree. Deleted try-except